src/services/__tests__/redisSessionStorage.test.ts

```diff
@@ -1396,6 +1398,89 @@ describe("RedisSessionStorage#userHasActiveSessions", () => {
   });
 });
 
+describe("RedisSessionStorage#isUserLogged", () => {
+  it("should return true if login type is LV", async () => {
+    mockGet.mockImplementationOnce((_, __) =>
+      Promise.resolve(
+        JSON.stringify({ a: anAssertionRef, t: LoginTypeEnum.LV })
+      )
+    );
+
+    const result = await sessionStorage.isUserLogged(aValidUser.fiscal_code);
+    expect(result).toEqual(E.right(true));
+  });
+
+  it("should return true if login type is LEGACY and user has an active session", async () => {
+    mockGet.mockImplementationOnce((_, __) =>
+      Promise.resolve(
+        JSON.stringify({ a: anAssertionRef, t: LoginTypeEnum.LEGACY })
+      )
+    );
+    mockSmembers.mockImplementationOnce((_) =>
+      Promise.resolve([`SESSIONINFO-${aValidUser.session_token}`])
+    );
+    mockGet.mockImplementationOnce((_, __) =>
+      Promise.resolve(
+        JSON.stringify({
+          createdAt: new Date(),
+          sessionToken: aValidUser.session_token,
+        })
+      )
+    );
+    mockGet.mockImplementationOnce((_, __) =>
+      Promise.resolve(JSON.stringify(aValidUser))
+    );
+
+    const result = await sessionStorage.isUserLogged(aValidUser.fiscal_code);
+    expect(result).toEqual(E.right(true));
+  });
+
+  it("should return false if no LollipopData was found", async () => {
+    mockGet.mockImplementationOnce((_, __) => Promise.resolve(null));
+
+    const result = await sessionStorage.isUserLogged(aValidUser.fiscal_code);
+    expect(result).toEqual(E.right(false));
+  });
+
+  it("should return false if login type is LEGACY and user has no active sessions", async () => {
```

Comment:
There is another scenario that should be tested. The SessionInfo has not an expire but the Session has it. We should test when the SMembers return a `SESSIONINFO` value and the next session get doesn't find a session.